src/api/properties/array.js

```diff
@@ -0,0 +1,14 @@
+export default {
+  coerce (val) {
+    return Array.isArray(val) ? val : [val];
+  },
+  default () {
+    return [];
+  },
+  deserialize (val) {
+    return val.split(',');
+  },
+  serialize (val) {
+    return val.join(',');
```

Comment:
But if we did that, then you'd have to escape double quotes, and it'd be a mess. I reckon for simple enum-style values, this'd be simpler.